cmd/gaia/app/app.go

```diff
@@ -149,7 +149,15 @@ func (app *GaiaApp) initChainer(ctx sdk.Context, req abci.RequestInitChain) abci
 	}
 
 	// load the initial stake information
-	stake.InitGenesis(ctx, app.stakeKeeper, genesisState.StakeData)
+	stake.InitGenesis(app.stakeKeeper, ctx, genesisState.StakeData)
```

Comment:
why did you switch it's standard that the context is always the first function input

Reply:
OK, updated all instances.